generators/client/templates/angular/src/main/webapp/content/scss/global.scss.ejs

```diff
@@ -151,39 +151,11 @@ Custom alerts for notification
 entity tables helpers
```

Comment:
Styles like below may not be relevant with custom theme (like material). It would be good to make those conditional and not apply if user chooses some theme

> .ng-valid[required],
> .ng-valid.required {
>     border-left: 5px solid green;
> }
> 
> .ng-invalid:not(form) {
>     border-left: 5px solid red;
> }

Reply:
I would keep them as it's good for validation, user can change it afterwards if they do not fit the theme

Reply:
I think the idea of pre-configured theme is to not apply any custom styles. If user require further customization, then, they should do manually.